book/src/book/book.controller.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { BookService } from './book.service';
+import { CreateBookDto } from 'src/common/dto/book.dto';
+
+@Controller('book')
+export class BookController {
+  constructor(private readonly bookService: BookService) {}
+
+  @Get()
+  async showAllBooks() {
```

Comment:
additionally you can type return value
```suggestion
  async showAllBooks(): Book[] {
```